github/files/drupal7/scripts/devops/reinstall/prepare_environment.yml

```diff
@@ -15,11 +15,19 @@
 - name: Set writable default dir
   sudo: yes
   file: dest=sites/default mode=777 state=directory recurse=yes
+  when: cmf == 'drupal'
```

Comment:
We need decision for how to implement different CMF/CMS
Currenlt there is a profile flow, that should be used only for Drupal
sql flow should be more flexible for ability to handle other CMF.
Here we should introduce varible for cmf name and possible even split files per CMF